engine/platform/src/platform_window_glfw3.cpp

```diff
@@ -19,7 +19,7 @@
 #include "platform_window_glfw3_private.h"
 #include "platform_window_opengl.h"
 
-#include <glfw/glfw3.h>
+#include <GLFW/glfw3.h>
```

Comment:
I think this was hidden somehow previously.. This hasn't changed anything between these versions, but I have to do it this way currently.